src/CTA.WebForms2Blazor/Factories/FileConverterFactory.cs

```diff
@@ -20,7 +18,7 @@ public class FileConverterFactory
         {
```

Comment:
We should eventually do as Jon suggested and replace these with categories such as markup files, coding files, images etc.. or we can also replace these with constants within those categories so we can have a word for each of these such as Constants.CSharpCodeFileExtension for .cs for example. This doe snot need to be done now we can just add it in  quip for todos for now.